app/utils/to-readable-date.ts

```diff
@@ -1,4 +1,4 @@
 export function toReadableDate(dateString: string): string {
```

Comment:
@bulentyusuf What do you think about changing the function to: 

```typescript
export function toReadableDate(dateString: string, locale = "de-DE"): string {
    const options: Intl.DateTimeFormatOptions = {year: "numeric", month: "long", day: "numeric"}
    return new Date(dateString).toLocaleDateString([locale], options)
}
```
This will keep the old default, but will leave it up to the user what format to choose.

Reply:
Sounds like a good solution.

Reply:
should I close the pull request?